### ISSUE TITLE
couldn't find file 'twitter/bootstrap'

### ISSUE THREAD

**Issue by jkarmel** [metaskills/less-rails-bootstrap#93]
My app can't seem to find the "twitter/bootstrap" file that this gem should be loading up. It's surprising because I recall a previous version working just great (though I am now using rails 4). I get the same issue when I run rake assets:precompile. 

I tried reinstalling this gem, as well as reinstalling less and less-rails. Still no luck. Any ideas about what I might be missing?

**Comment by mchavarriagam** [comment]
I'm having the same issue in my production environment (Debian GNU/Linux 6.0) with Ruby 2.1.2, Rails 4.1.8, less-rails 2.2.3 and less-rails-bootstrap 2.1.1.

This is what my Gemfile looks like for assets:

```
group :assets do
  gem 'coffee-rails', '4.1.0'

  gem 'therubyracer', '0.12.1', :require => 'v8'

  gem 'less-rails', '~> 2.3.1'
  gem 'less-rails-bootstrap', '~> 2.3.3'
end
```

This issue appeared when I moved to Rails 4 (coming from Rails 3.x) and tried to do rake assets:precompile.

The error I'm getting is -> Sprockets::FileNotFound: couldn't find file 'twitter/bootstrap/transition' (I have //= require twitter/bootstrap/transition in the application.js file).

I followed the issues guide (rake tmp:clear, deleted /public/assets, etc). Any ideas? Thanks!
